setup.py

```diff
@@ -62,7 +63,7 @@
 ]
 
 tutorials_require = [
-    'scikit-learn>=0.22,<0.23',
+    'scikit-learn>=0.24,<0.25',
```

Comment:
How did you determine this range?

Reply:
I think `0.24` was the minimum for python 3.9. I just kept it at a one version gap

Reply:
From https://pythonrepo.com/repo/scikit-learn-scikit-learn-python-deep-learning:
```
0.24.0(Dec 22, 2020)
We're happy to announce the 0.24 release. You can read the release highlights under https://scikit-learn.org/stable/auto_examples/release_highlights/plot_release_highlights_0_24_0.html and the long version of the change log under https://scikit-learn.org/stable/whats_new/v0.24.html#version-0-24-0

This version supports Python versions 3.6 to 3.9.
```

Reply:
What about 1.0 and above @amontanez24 ?

Reply:
I haven't tried it. They are definitely Python 3.9 compatible, I just wasn't sure if it would cause anything to crash

Reply:
I think they mentioned that the 1.0 was not really a "big" change, so 1.0 was kind of equivalent to what a 0.25 would have been, and that they only changed it to 1.0 because they thought that the library was mature enough.
So I would say that if we extend it to `<2` and all the tests pass, we can go ahead with it.